packages/devtools_app/lib/src/shared/side_panel.dart

```diff
@@ -271,11 +288,23 @@ class ReleaseNotesController {
   }
 
   void toggleReleaseNotesVisible(bool visible) {
-    _releaseNotesVisible.value = visible;
+    _isVisible.value = visible;
   }
 
   String _releaseNotesUrl(String currentVersion) {
     return '$_flutterDocsSite/development/tools/devtools/release-notes/'
         'release-notes-$currentVersion-src.md';
   }
 }
+
+class SidePanelControllerMarkdownString extends SidePanelController {
```

Comment:
do we need an additional class for this? why not just add the `markdownText` setter to SidePanelController directly?